sample/simple/cmd/app.go

```diff
@@ -26,15 +26,39 @@ package cmd
 
 import (
 	"errors"
+	"os"
+	"os/user"
 	"strings"
 	"time"
 
 	"github.com/desertbit/grumble"
+	"github.com/fatih/color"
 )
 
 var App = grumble.New(&grumble.Config{
 	Name:        "foo",
 	Description: "An awesome foo bar",
+	PromptRuntime: func() string {
+		username := "NO-USER"
+		hostname := "NO-HOSTNAME"
+		user, err := user.Current()
+		if err == nil {
+			username = user.Name
```

Comment:
On my system, `Name` is empty. Using `Username` yields the correct name.

Reply:
This runtime function just implement as sample, i will fix it later.